frontend/io-webapp/src/connect/LogIn.js

```diff
@@ -9,17 +9,17 @@ const LogIn = ({ state, dispatch }) => {
     e.preventDefault();
 
     (async () => {
-      await fetch("http://localhost:8080/login", {
+      await fetch(process.env.REACT_APP_BACKEND_URL + "/login", {
```

Comment:
Czyli robiąc na froncie jedyna różnica jest taka, że korzystamy z tej zmiennej zamiast bezpośrednio z adresu? Robiąc coś na osobnym branchu to właściwie nic nie zmienia i korzystamy dalej z tego co mamy lokalnie w kontenerze i jedyne co to zmienia to pozwala na działanie serwera, czy coś źle rozumiem?

Reply:
tak, jedyne co się zmienia to korzystanie z tej zmiennej